modules/widgets/social-media-icons.php

```diff
@@ -115,15 +107,11 @@ public function widget( $args, $instance ) {
 		if ( ! empty( $instance['vimeo_username'] ) ) {
 			$html .= '<li><a title="' . sprintf( $alt_text, esc_attr( $instance['vimeo_username'] ), 'Vimeo' ) . '" href="' . esc_url( 'https://vimeo.com/' . $instance['vimeo_username'] . '/' ) . '" class="genericon genericon-vimeo" target="_blank"><span class="screen-reader-text">' . sprintf( $alt_text, esc_html( $instance['vimeo_username'] ), 'Vimeo' ) . '</span></a></li>';
 		}
-
-		/**
-		 * Fires at the end of the list of Social Media accounts, inside the unordered list.
-		 *
-		 * Can be used to add a new Social Media Site to the Social Media Icons Widget.
-		 *
-		 * @since 3.7.0
-		 */
-		do_action( 'jetpack_social_media_icons_widget_list_after' );
+		
+		if ( ! empty( $instance['google_username'] ) ) {
+			$title_text = (is_numeric( $instance['google_username'] ) ? $alt_text_numeric : $alt_text);
+			$html .= '<li><a title="' . sprintf( $title_text, esc_attr( $instance['google_username'] ), 'Google+' ) . '" href="' . esc_url( 'https://plus.google.com/u/1/' . $instance['google_username'] . '/' ) . '" class="genericon genericon-googleplus" target="_blank"><span class="screen-reader-text">' . sprintf( $alt_text, esc_html( $instance['google_username'] ), 'Google+' ) . '</span></a></li>';
```

Comment:
I just checked - it doesn't matter whether it's a 0 or 1, that's based on how many accounts you have signed in. If it gets changed to a 0 it will still go to the same page.
I also replaced it with my old one, +BrendanCarr84, and it went to the correct page.